settings.gradle

```diff
@@ -1,2 +1,2 @@
-include ':library', ':sample', ':stub-android'
+include ':library', /*':sample',*/ ':stub-android'
```

Comment:
nit: Remove comment.

Reply:
Good questinon, we need to decide whether to invest some time in reanimating this sample or just throw it away. I was thinking of second option, but if there's a chance to make a contribution to the library, maybe we need to take first option.

Reply:
I think we can make as handy for us firstly. And publish if we will want.